core-helm-chart/values.yaml

```diff
@@ -2,5 +2,12 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+# name of control plane to set as WDS
 ControlPlaneName: wds1
 
+# list of allowed resource groups, comma separated.
```

Comment:
I would say "Kubernetes API groups" instead of "resource groups". Both because it uses standard Kubernetes terminology and because it does not use a term that means something else in everybody's cloud.

Reply:
done